pyPhin.py

```diff
@@ -1,19 +1,26 @@
+#!/usr/bin/env python3
+"""
+pHin Unoffical Python API
```

Comment:
As some comments about pHin (eg, pHin Smart Water Monitor and point to the website)